### DISCUSSION TITLE
An attempt was made to load a program with an incorrect format

### DISCUSSION THREAD

**Discussion by stromhaug-tommy** [Macad3D/Macad3D#17] [Q&A] [answered]
Hi,

System.BadImageFormatException: 'Could not load file or assembly 'Macad.Occt, Version=2.0.1353.3, Culture=neutral, PublicKeyToken=f3b86e0df7940e9a'. An attempt was made to load a program with an incorrect format

I get this error when trying to run an application that only uses the wrapper code.
It is set to x64. What could be the problem. I use the MACAD project as entry point for ny code.

**Comment by stromhaug-tommy** [comment]
I have double checked the exe file and it is a x64. Occt is also x64.

**Comment by dipts** [comment, marked as answer]
Hi @stromhaug-tommy,

this exception is thrown whenever a module cannot be loaded. In addition to the wrong target format, it is possible that a dependency was not found. You can check that with [Dependency Walker](https://www.dependencywalker.com/) (on Macad.Occt).

**Reply by stromhaug-tommy** [reply]
Thank you very much for your answer! I used https://github.com/lucasg/Dependencies, and found the missing dlls :) 
Thank you very much for this great wrapper!
